src/test_storageiface.py

```diff
@@ -0,0 +1,139 @@
+'''
```

Comment:
Should we have the tests in src, or maybe keep them in a test folder?

Reply:
It could indeed be moved, along with its `wopiserver-test.conf` configuration